numpy/ctypeslib.py

```diff
@@ -97,10 +97,10 @@ def load_library(libname, loader_path):
 
         Parameters
         ----------
-        libname : str
+        libname : path-like
             Name of the library, which can have 'lib' as a prefix,
             but without an extension.
-        loader_path : str
+        loader_path : path-like
             Where the library can be found.
```

Comment:
Can you add a`..versionchanged` somewhere?

Reply:
Will this need a release note as well?
I figure the change was small enough, but it can't hurt double checking.

Reply:
I'd probably write a short one, yes. No need to hit the mailing list.

Reply:
Let's do it; `versionchanged` was added in https://github.com/numpy/numpy/pull/17530/commits/3271a08baf9500f0112be24e7573d9ae2945f847 and the release note in https://github.com/numpy/numpy/pull/17530/commits/d690cf60beef0739f6d7c15e778c37e528afe58f.